dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageChecker.java

```diff
@@ -132,6 +132,8 @@ private static void printStaticLinkageErrors(
   /**
    * Parses arguments to instantiate the class with configuration specified in arguments.
    *
+   * TODO(suztomo): create a class to represent command line option #209
```

Comment:
These TODOs inside Javadoc won't look good when rendered. Unless they're TODOs _about the Javadoc_, I'd suggest moving them below the Javadoc and making them `// TODO` comments.

Reply:
Removed TODOs from Javadoc